src/tests/ftest/util/server_utils.py

```diff
@@ -77,15 +77,18 @@ def create_server_yaml(basepath):
         ServerFailed: if there is an reading/writing yaml files
 
     """
+    with open(os.path.join(basepath, ".build_vars.json")) as json_vars:
+            build_vars = json.load(json_vars)
+    
     # Read the baseline conf file data/daos_server_baseline.yml
     try:
-        with open('{}/{}'.format(basepath, DEFAULT_FILE), 'r') as read_file:
+        with open('{}/{}'.format(build_vars["PREFIX"], DEFAULT_FILE), 'r') as read_file:
```

Comment:
If using relative paths:
```suggestion
        with open(default_yaml, 'r') as read_file:
```